examples/value.rs

```diff
@@ -0,0 +1,40 @@
+// Copyright 2015 The Noise-rs Developers.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! An example of using value noise
+
+extern crate noise;
+
+use noise::{value2, value3, value4, Seed, Point2};
+
+mod debug;
+
+fn main() {
+    debug::render_png("value2.png", &Seed::new(0), 1024, 1024, scaled_value2);
+    debug::render_png("value3.png", &Seed::new(0), 1024, 1024, scaled_value3);
+    debug::render_png("value4.png", &Seed::new(0), 1024, 1024, scaled_value4);
+    println!("\nGenerated value2.png, value3.png and value4.png");
```

Comment:
I figured as much. I'll rebase on top of master as part of my final squash
commit.
On Jun 27, 2016 9:04 AM, "Brendan Zabarauskas" <notifications@github.com>
wrote:

> In examples/value.rs
> <https://github.com/bjz/noise-rs/pull/109#discussion_r68572101>:
>
> > +// See the License for the specific language governing permissions and
> > +// limitations under the License.
> > +
> > +//! An example of using value noise
> > +
> > +extern crate noise;
> > +
> > +use noise::{value2, value3, value4, Seed, Point2};
> > +
> > +mod debug;
> > +
> > +fn main() {
> > +    debug::render_png("value2.png", &Seed::new(0), 1024, 1024, scaled_value2);
> > +    debug::render_png("value3.png", &Seed::new(0), 1024, 1024, scaled_value3);
> > +    debug::render_png("value4.png", &Seed::new(0), 1024, 1024, scaled_value4);
> > +    println!("\nGenerated value2.png, value3.png and value4.png");
>
> @Razaekel <https://github.com/Razaekel>: note that this is reintroducing
> the log message to an example - you'll have to fix that in #104
> <https://github.com/bjz/noise-rs/pull/104> once if we merge this first
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/bjz/noise-rs/pull/109/files/75b2622978a491e76b20a3c2583236ba61279ba9#r68572101>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AEC6Xrl5UoXTL6Omfwe8_RJC_Cc9MJOgks5qP8phgaJpZM4I-iPE>
> .
>